response.proto

```diff
@@ -172,6 +177,7 @@ message Journey {
     optional uint64 min_waiting_dur = 18;
     optional uint32 nb_vj_extentions = 19;
     optional uint32 nb_sections = 20;
+    optional Durations durations = 21;
```

Comment:
journey.duration should be removed in the protobuf (but journey.durations.total should be used in the json to populate journey.duration for backward compatibility).